backend/src/v5/middleware/dataConverter/inputs/users.js

```diff
@@ -183,33 +183,17 @@ const generateSignUpSchema = () => {
 					try {
 						await getUserByUsername(value, { _id: 1 });
 						return false;
-					} catch {
-					// do nothing
-					}
-				}
-				return true;
-			}).required(),
-		email: types.strings.email.test('checkEmailAvailable', 'Email already exists',
-			async (value) => {
-				if (value) {
-					try {
-						await getUserByQuery({ 'customData.email': value }, { _id: 1 });
-						return false;
 					} catch {
 						// do nothing
 					}
 				}
 				return true;
 			}).required(),
-		password: types.strings.password.required(),
-		firstName: types.strings.name.required().transform(formatPronouns),
-		lastName: types.strings.name.required().transform(formatPronouns),
 		countryCode: types.strings.countryCode.required(),
 		company: types.strings.title.optional(),
 		mailListAgreed: Yup.bool().required(),
 		...(captchaEnabled ? { captcha: Yup.string().required() } : {}),
-	})
-		.noUnknown().required();
+	}).noUnknown().required();
 
 	return captchaEnabled
```

Comment:
Can we strip the captcha field after testing it, then you can just pass the whole body into the state for sso endpoints without worry